code/__defines/research.dm

```diff
@@ -1,14 +1,16 @@
 #define SHEET_MATERIAL_AMOUNT 2000
 #define SHEET_UNIT "<small>cm<sup>3</sup></small>"
 
+#define REAGENT_UNITS_PER_MATERIAL_UNIT (REAGENT_UNITS_PER_MATERIAL_SHEET / SHEET_MATERIAL_AMOUNT)
 #define REAGENT_UNITS_PER_MATERIAL_SHEET 20
 #define REAGENT_UNITS_PER_GAS_MOLE 10
-#define MATTER_UNITS_PER_REAGENT_UNIT (REAGENT_UNITS_PER_MATERIAL_SHEET / SHEET_MATERIAL_AMOUNT)
 
 #define MATTER_AMOUNT_PRIMARY       SHEET_MATERIAL_AMOUNT
 #define MATTER_AMOUNT_REINFORCEMENT (MATTER_AMOUNT_PRIMARY * 0.5)
 #define MATTER_AMOUNT_TRACE         (MATTER_AMOUNT_PRIMARY * 0.1)
 
+#define GENERIC_SMELTING_HEAT_POINT 5000
```

Comment:
an odd place for these defines, why not materials or chemistry defines?

Reply:
Because I was looking at this file anyway and it didn't seem to matter.
I can move them if you really want but placement is pretty arbitrary.

Reply:
this file is half materials anyway so eh fine either way